Update artifact configuration page (#39)
gatling/frontline-cloud-doc: content/reference/user/artifact_conf/index.md

> To access the Artifacts section, click on **Artifacts** in the navbar.

Reviewer: in the SideNav, not the navbar

Author: Isn't the side bar called the "navigation bar" elsewhere in the docs? (Either way, neither "SideNav" nor "navbar" seem appropriate terminology for the user facing docs.)

Reviewer: Sidenav is the one on the left side, and navbar at the top side. Indeed, the naming can get a bit confusing, and may need a change

Author: I checked, we do seem to call it navigation bar everywhere else in the docs. I'm going to use that; if it's not correct then it should be changed globally.

{{< img src="artifact-upload-empty.png" alt="Artifact upload empty" >}}

> You can then drag and drop your artifact to filled the form and proceed to upload.

Reviewer: on line 38, can you modify: [Artifact Generation documentation]()

{{< img src="artifact-upload-filled.png" alt="Artifact upload filled" >}}

> You can upload multiple artifact concurrently. Upload progress will be shown in the Artifacts table.

Reviewer 2: ```suggestion
You can upload multiple artifacts concurrently. Upload progress will be shown in the Artifacts table.
```

Reviewer 2: Active voice
```suggestion
You can upload multiple artifact concurrently. The Artifacts table will show the upload progress.
```

> You can upload multiple artifacts concurrently. Upload progress will be shown in the Artifacts table.

Reviewer: Upload**s** progress

> An upload in progress can be cancelled by clicking on the cancel icon

Reviewer 2: Active voice
```suggestion
You can cancel an upload in progress by clicking on the cancel icon.
```

You'll need:
* an [API token]({{< ref "../../admin/api_tokens" >}}) with at least the `Artifacts` permission

> * the Artifact's ID, which you can copy from the WebUI.

Reviewer: , which can be copied from the artifacts table.

Reviewer 2: Always prefer the active voice!
